src/AzureBlobStorageUpdateSession/UpdateSession/UpdateDomainEntity.cs

```diff
@@ -0,0 +1,22 @@
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Etg.Yams.Azure.UpdateSession
+{
+    public class UpdateDomainEntity : TableEntity
+    {
+        public UpdateDomainEntity()
```

Comment:
Where is this used?

Reply:
This represents a row in the Azure Table, it's used for all Yams rows

Reply:
Sorry, I meant the empty constructor, can it be removed?

Reply:
Ah, a default constructor is required by Azure Table client. Given that there is no other constructors, it can probably be removed (it's implicit).